### ISSUE TITLE
`Physical Review. E` -> `Phys. Rev.,`

### ISSUE THREAD

**Issue by njzjz** [pierre-24/pyiso4#13]
`E` is missing!

```py
>>> from pyiso4.ltwa import Abbreviate
>>> abbreviator = Abbreviate.create()
>>> abbreviator("Physical Review. E", remove_part=True)
'Phys. Rev.,'
```

**Comment by klb2** [comment]
Closed in 3c439bc (045bcd1)

**Comment by njzjz** [comment]
Thank you!
